scripts/mmti-external-links/update-major-mines.js

```diff
@@ -0,0 +1,138 @@
+// @ts-check
+'use strict';
+
+var mongodb     = require('mongodb');
+var Promise     = require('promise');
+var _           = require('lodash');
+var uniqueBy    = require('./utils').uniqueBy;
```

Comment:
While correct, is not necessary due to the require of lodash.  Lodash has a method _.uniqBy that achieves the same result as the code in the utils file.